lib/store/cleanup.go

```diff
@@ -30,19 +31,15 @@ import (
 
 // CleanupConfig defines configuration for periodically cleaning up idle files.
 type CleanupConfig struct {
-	Disabled            bool          `yaml:"disabled"`
-	Interval            time.Duration `yaml:"interval"`             // How often cleanup runs.
-	TTI                 time.Duration `yaml:"tti"`                  // Time to idle based on last access time.
-	TTL                 time.Duration `yaml:"ttl"`                  // Time to live regardless of access. If 0, disables TTL.
-	AggressiveThreshold int           `yaml:"aggressive_threshold"` // The disk util threshold to trigger aggressive cleanup. If 0, disables aggressive cleanup.
-	AggressiveTTL       time.Duration `yaml:"aggressive_ttL"`       // Time to live regardless of access if aggressive cleanup is triggered.
+	Disabled             bool          `yaml:"disabled"`
+	Interval             time.Duration `yaml:"interval"`               // How often cleanup runs.
+	TTI                  time.Duration `yaml:"tti"`                    // Time to idle based on last access time.
+	TTL                  time.Duration `yaml:"ttl"`                    // Time to live regardless of access. If 0, disables TTL.
+	ExcludeOtherServices bool          `yaml:"exclude_other_services"` // Whether to exclude other services from the disk util calculation.
```

Comment:
@Anton-Kalpakchiev  why are we making this config-driven?
In my opinion, Kraken's cache cleanup should always exclude other services from the disk utilization calculation. I don’t see a scenario where including other services’ disk utilization would improve the cache cleanup behavior.

Reply:
The biggest reason is backward compatibility.